### ISSUE TITLE
aria2c is not running on port 6800

### ISSUE THREAD

**Issue by bmndan** [Kyle-Kyle/baidudl#85]
Upon clicking the RPCButton, I get this error reading: aria2c is not running on port 6800
What could be the issue?
I have used method 1 (installed Chrome extension)

**Comment by Kyle-Kyle** [comment]
you need to download baidudl-rpc first and run it on background.

**Comment by Kyle-Kyle** [comment]
https://github.com/Kyle-Kyle/baidudl/tree/master/baidudl_rpc

**Comment by bmndan** [comment]
Ok, following those steps, I now don't have the error message, but just a "Ready" message
Here's what I tried:
1. downloaded aria2c.exe and baidudl_rpc.bat, put in same folder
2. run baidudl_rpc.bat
3. enter some folder in pan.baidu.com
4. click the blue lightbulb icon in chrome
5. it then shows the available files in the folder
6. I click RPCDownload, but nothing happens (message is "Ready.")

**Comment by Kyle-Kyle** [comment]
is baidudl_rpc.bat really running? btw... the master branch is abandoned, it will not be maintained. you could try [wip] branch which currently should work on share page.

**Comment by bmndan** [comment]
Yep, I first open baidudl_rpc.bat, then enter the pan.baidu folder URL, and click RPCDownload (while baidudl_rpc.bat is running), it just says "Ready"
by the way when I click "Generate" it says "Error: cant't share this file"

**Comment by Kyle-Kyle** [comment]
@bmndan you are using the wrong extension.
https://chrome.google.com/webstore/detail/baidu-dl/lflnkcmjnhfedgibjackiibmcdnnoadb

**Comment by Kyle-Kyle** [comment]
that was the old one. Due to some reasons I can not maintain it any longer

**Comment by Kyle-Kyle** [comment]
BTW. I recommend you to use the github version. [wip] branch is better.

**Comment by bmndan** [comment]
Ok, it seems a newer version indeed
In this one I get the message "Login first"
Do I have to have Baidu account to use it?

**Comment by Kyle-Kyle** [comment]
well... for now.. yes. that's because I'm currently working on another branch.

**Comment by Kyle-Kyle** [comment]
and the version 1 is not maintained.

**Comment by bmndan** [comment]
Oh, I had a feeling of that, I just haven't a Chinese phone number to register on baidu
It would be really great being able to use it without an account, I think a lot of people in need of it
